sass/components/_suggestion-list.scss

```diff
@@ -109,6 +109,15 @@
         max-height: 205px;
         width: 100%;
     }
+
+    span
+        .status
+            .offline--icon,
+            .online--icon,
+            .away--icon,
+            .dnd--icon {
+                margin: auto 7px auto 7px;
```

Comment:
can be shortened to `auto 7px`

Reply:
thanks for the tips didn't know this one 👍